docs/howto/manage-tracks.rst

```diff
@@ -3,7 +3,7 @@
 Manage tracks
 =============
 
-..    See also: :ref:`track` Add link to Juju docs > charm > channel > track
+    See first: :external+juju:ref:`Juju | Charm channel track <track>`
```

Comment:
(Oof, GitHub ate this out of my review.)

```suggestion
    See first: :external+juju:ref:`Juju | Charm channel track <channel>`
```

According to the current build, the target is [`channel`](https://github.com/juju/juju/blob/b85a5723766d1e5c126135ccbdcff897c6307b22/docs/user/reference/charm/channel.md?plain=1#L1). Is there an upcoming PR that will change it to `track`?

Reply:
Yes. The same goes for `guardrail` below. We could leave it as `channel` in both cases but `track`/`guardrail` would help people jump to the specific section so it would be more helpful.